.github/workflows/main.yml

```diff
@@ -118,3 +118,28 @@ jobs:
     - uses: dtolnay/rust-toolchain@stable
     - name: Run Clippy
       run: cargo clippy --all-targets --all-features -- -D warnings
+  CheckExternalTypes:
+    name: check-external-types (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
+        rust:
+          # `check-external-types` requires a specific Rust nightly version. See
+          # the README for details: https://github.com/awslabs/cargo-check-external-types
+          - nightly-2024-06-30
```

Comment:
The readme mentions `nightly-2025-05-04` here: https://github.com/awslabs/cargo-check-external-types?tab=readme-ov-file#how-to-use. But I assume this version will also work.

Reply:
It seems like the releases page has the recent version: https://github.com/awslabs/cargo-check-external-types/releases.